src/main/java/io/strimzi/kafka/bridge/http/HttpSourceBridgeEndpoint.java

```diff
@@ -72,7 +73,7 @@ public void handle(Endpoint<?> endpoint) {
 
         String topic = routingContext.pathParam("topicname");
 
-        List<KafkaProducerRecord<K, V>> records;
+        List<KafkaProducerRecord<K, V>> records = new ArrayList<>();
```

Comment:
MAbe I don't see the whole code ... but isn't it better to just for example return when `messageConverter == null` instead of creating the emepty arraylist which is never used?